linetools/isgm/abssurvey.py

```diff
@@ -253,7 +251,7 @@ def ions(self, iZion, skip_null=False):
         Table of values for the Survey
         """
         keys = [u'name', ] + self.abs_sys()[0]._ionN.keys()
-        t = copy.deepcopy(self.abs_sys()[0]._ionN[0:1])
+        t = Table(copy.deepcopy(self.abs_sys()[0]._ionN[0:1]))  # Avoids mixin trouble
```

Comment:
Well, this is just copying a snipped of an astropy Table to
get the rows.  Maybe there is a method in that already..